### DISCUSSION TITLE
Choose or add a "type" when importing GPX waypoints into favourites

### DISCUSSION THREAD

**Discussion by Busterama452** [osmandapp/OsmAnd#11364] [Q&A] [answered]
# 🚀 feature request

### Description

As a user of OSMAND I would like to be able to categorize waypoints that I import from GPX files into my favorites so that I can keep my favorites organized.

I can currently import waypoints from a GPX, but they're loading into Favorites without any categorization.   When importing rather large quantities of waypoints, it would be nice to be able to import them all into a single category/type rather than just dumped in as generic favorites.

**Comment by pebogufi** [comment, marked as answer]
It is very easy to get what you want. After import of a number of waypoints whithout type you find them under favourite group name "favourites". Just rename that group to a name, you like.

**Comment by Busterama452** [comment]
> It is very easy to get what you want. After import of a number of waypoints whithout type you find them under favourite group name "favourites". Just rename that group to a name, you like.

Awesome work around.   Thank you!
